src/jit/block.cpp

```diff
@@ -213,7 +213,7 @@ void BasicBlock::dspBlockILRange()
     }
     else
     {
-        printf("[???..");
+        printf("[???" "..");
```

Comment:
Right, clang handled the '???)' as a trigraph and generated ']'. At line 216, it is not needed, but I've added it based on a feedback from @jkotas who found it nicer to be symmetrical with the place below where it was needed.
I will add a comment there.